guacamole/src/main/frontend/src/translations/de.json

```diff
@@ -580,9 +693,15 @@
         "FIELD_HEADER_RECORDING_INCLUDE_KEYS"   : "Tastendrücke aufzeichnen:",
         "FIELD_HEADER_RECORDING_NAME" : "Aufzeichnungsname:",
         "FIELD_HEADER_RECORDING_PATH" : "Aufzeichnungspfad:",
+
```

Comment:
I think this extra space can be removed - the rest of the `FIELD_HEADER` items don't seem to be broken up?

Reply:
I removed it. Rest seems fine yeah